### PR TITLE
Features #401 and #402

### PR DESCRIPTION
**Pull request by amenk**

### CODE REVIEW

**Comment by davidjbradshaw** [comment]
Would you be able to update the docs as well.

**Comment by amenk** [comment]
no today - would be gread if you could do it :-) *lazyPR*

2016-09-01 15:19 GMT+02:00 David J. Bradshaw <notifications@github.com>:

**Comment by davidjbradshaw** [comment]
Came up with a simpler fix for #401, Still thinking about #402

File: js/iframeResizer.contentWindow.js [hunk 1]
```diff
@@ -198,6 +199,8 @@
 		inPageLinks.enable = (undefined !== data[12]) ? strBool(data[12]): false;
 		resizeFrom         = (undefined !== data[13]) ? data[13]         : resizeFrom;
 		widthCalcMode      = (undefined !== data[14]) ? data[14]         : widthCalcMode;
+		customSelector     = (undefined !== data[15]) ? data[15]         : customSelector;
+		console.log(data);
```
Comment (wardpeet) [inline js/iframeResizer.contentWindow.js]: remove debug code please

**Comment by amenk** [comment]
Done

**Comment by dKard** [comment]
How can use this script on WordPress pages and post? I don't have coding experience.

**Pull request closed without merging**